Topics/Teamwork.md

```diff
@@ -1 +1,15 @@
 ## Resources for Teamwork
+
+
+### Using Kanban Boards
+
+Whether its GitHub Projects, Trello, Asana or Jira, if you're working with a software development team, there's a good chance you will be working with a Kanban board for your project management.
+
+Using them effectively as a team is a combination of keeping things clear for yourself, and clear for others.
+
+Kanban boards are different from a personal todo-list in that they are shared with a team and updated all the time. This all-encompassing nature has the advantage that anyone can check the board and see what is being worked on and at what stage, but has the pre-requisite that of clear wording and being constantly updated to ensure clarity. Setting [SMART goals](https://www.atlassian.com/blog/productivity/how-to-write-smart-goals) become essential, ensuring everyone else has a clear idea of what you're working on.
+
+Overall, with a bit more emphasis on logging and tracking, Kanban boards provide the perfect medium for a team to collaborate on. As a final note, remember that these are just tools to supplement collaboration. Some things can't be replaced by a quick conversation or a direct message. For a few more resources on using Kanbans with a team, see the following resources:
+
+- [Kanban methodology for software development by Atlassian, the creators of Jira](https://www.atlassian.com/agile/kanban)
+- [How Kanban Methodology Can Improve Your Team (as well as a history lesson on Toyota, the "creators" of Kanban cards)](https://www.lucidchart.com/blog/how-kanban-methodology-can-improve-your-team)
```

Comment:
I think the order of these bottom two links should be switched - the second one can serve as an intro and motivating example, whereas the first one is more in depth.

Reply:
That makes sense, appreciate the look-over.